src/core/plugins/core-commands/server/commands/moderator/vehicle.ts

```diff
@@ -17,12 +18,15 @@ class VehicleCommands {
 
     @command('repairVehicle', '/repairVehicle - Repairs an vehicle by administrative power.', PERMISSIONS.ADMIN)
     private static repairVehicleCommand(player: alt.Player) {
-        if (!player.vehicle || !player.vehicle.data) {
+        if (!player.vehicle) {
+            return;
+        }
+        Athena.vehicle.funcs.repair(player.vehicle);
+        if (!player.vehicle.data) {
```

Comment:
This if statement can go away if you want to enable it for temp vehicles if i am not mistaken.

Reply:
No because temp vehicles do not have `data` set